ava-labs/core-mobile
# CP-8430: Poll feature flags from proxy-api

## Description

**Ticket: [CP-8430]** 

* Implement version-specific feature flags as described in [this pull request](https://github.com/ava-labs/extension-avalanche/pull/1205) for the extension.
* Implement feature flags polling from the proxy API as detailed in [this pull request](https://github.com/ava-labs/extension-avalanche/pull/1387) for the extension.

## Testing
* Add `testing-version-specifics` to `FeatureGates` 
* Go to [testing-version-specifics flag](https://us.posthog.com/project/23713/feature_flags/41776) configured in Posthog
* Try toggling the feature on and off, and experiment with the payload by using a proper semver range to see how it affects the flag's state

[CP-8430]: https://ava-labs.atlassian.net/browse/CP-8430?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
